packages/playground/.nvmrc

```diff
@@ -0,0 +1 @@
+10.13.0
```

Comment:
I'm on 10.5.0. Why did you choose 10.13.0?

Reply:
@snario 10.13.0 is the standard LTS version. We should be working on top of that.

Reply:
Okay